tests/contrib/forecast/test_forecaster.py

```diff
@@ -103,17 +103,24 @@ def model(self, zero_data, covariates):
 @pytest.mark.parametrize("t_obs", [1, 7])
 @pytest.mark.parametrize("t_forecast", [1, 3])
 @pytest.mark.parametrize("batch_shape", [(), (4,), (3, 2)], ids=str)
-@pytest.mark.parametrize("cov_dim", [0, 1, 6])
+@pytest.mark.parametrize("cov_dim", [0, 1, 5])
 @pytest.mark.parametrize("obs_dim", [1, 2])
 @pytest.mark.parametrize("dct_gradients", [False, True])
 @pytest.mark.parametrize("Model", [Model0, Model1, Model2, Model3, Model4])
-def test_smoke(Model, batch_shape, t_obs, t_forecast, obs_dim, cov_dim, dct_gradients):
+@pytest.mark.parametrize("engine", ["svi", "hmc"])
```

Comment:
`hmc` test is slow (~3m in my system) :(

Reply:
can we specify fewer samples and shorter warmup etc. to speed it up?

Reply:
awesome! Time is reduced to 30s now. Thanks a lot!